src/bin/dg2el.ts

```diff
@@ -0,0 +1,65 @@
+import * as es from 'elasticsearch'
```

Comment:
This is superfluous but an example of a quick and dirty way of gettting dgraph -> elasticsearch